.github/workflows/build.yaml

```diff
@@ -32,7 +32,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
should we keep building on 22.04 also?
we can use a reference to the strategy.matrix as the runs-on value here, e.g. https://github.com/project-machine/atomfs/blob/main/.github/workflows/build.yaml#L7

Reply:
we don't particularly care about 24.04 right? final stacker is a static binary

Reply:
I didn't think we need to keep building on both; working on 24.04 will last longer than 22.04 will.

Reply:
ok

Reply:
I think if we can figure out a way to test building on both 22.04 and 24.04 weekly, that would be best to support the dev case where community folks want to build/debug stacker; versus requiring all PRs to do both builds passing;

thoughts?